product-form.html

```diff
@@ -100,9 +100,9 @@ <h2>Qualities of Your Coin</h2>
                     </div>
                 </fieldset>
 
-                <p></p>
+                <p>
                     <label for="condition">What condition is your coin in?</label>
-                    <input type="range" name="condition" id="condition" min="1" max="10" required>
+                    <input id="condition" type="range" name="condition" min="1" max="10" required>
                     <span id="condition-display">5</span>
                 </p>
```

Comment:
`<div>` tags are used for styling- CSS. Start using more semantic HTML like a `<section>` or `<article>` tag.

Reply:
Which lines exactly?

Reply:
instead of your `<p>`
Line 106 and it's matching opening tag